packages/cli/README.md

```diff
@@ -273,6 +273,30 @@ bubblewrap fingerprint generateAssetLinks <flags>
 Flags:
  - `--output=<filename>`: path from where to load the project configuration.
 
+## `play`
+
+:warning: This is an experimental feature.
+
+Manages the artifacts for your Google Play Project.
+
+Usage:
+
+```
+bubblewrap play --serivceAccountFile="/path/to/serivce/account"  --retain 86 --publish
+```
+
+This tool requires a service account file to work correctly. Please see [this documentation](https://github.com/chromeos/pwa-play-billing#setup-a-service-account) for setting up a service acocunt.
+
+Options:
+  - `--publish`: publishes the prebuilt file to the Google Play Store under the internal track or specificed track.
+  - `--serviceAccountJsonFile`: sets the service acocunt json file location in the twa-manifest.
+  - `--manifest`: specifies the manifest file to use if not in the current directory.
+  - `--appBundleLocation`: specifies the location of the appbundle to upload to Google Play.
+  - `--targetDirectory`: the directory that versionCheck should run in.
+  - `--versionCheck`: checks that the version specified is higher than the one currently in the Google Play Store.
+  - `--retain`: specifies the bundles to retain for release (this would be Android only bundles if releasing a Chrome OS only release).
+  - `--removeRetained`: removes the specified bundle if no longer relevant.
+  - `--listRetained`: shows a list of existing retained bundles.
```

Comment:
Can we split this out into a different command?

Reply:
So something like `bubblewrap retained --${command}`?

Reply:
Thinking about it a bit more, it seems that we have one `bubblewrap play` command that can do a bunch of different things. In fact most of the bubblewrap commands are `bubblewrap <verb>` (such as `bubblewrap build`, `bubblewrap update`, `bubblewrap validate`) which I think makes sense. `bubblewrap play` on the other hand doesn't tell us what the command is going to do (it connects to Play and does what?).

It would be good to split out which of these options determines what `bubblewrap play` actually does, and which are just configuration options. I think maybe it would work better to have something like the following commands:

```
bubblewrap playPublish
bubblewrap playListRetained
bubblewrap playRemoveRetained
```

And then each of these commands would have `serviceAccountFile`, `manifest`, etc options.

Bear in mind that I can't quite remember 100% what each of these options does - so maybe `playRemoveRetained` actually doesn't do anything on it's own and having a `--removeRetained` option for `playPublish` makes sense. Similarly, maybe `versionCheck` could be a separate command instead of a flag for the `playPublish` command.

Reply:
I still think we should try to simplify the `playPublish` command if possible, but yeah, I think that looks like a good path forward.

Reply:
Ok, cool. I will go and update the docs in this CL and then do a few small CLs to cover the playAction commands using the docs as a guide.